lib/api/apiUtils/object/createAndStoreObject.js

```diff
@@ -200,11 +200,12 @@ function createAndStoreObject(bucketName, bucketMD, objectKey, objMD, authInfo,
                     const md5 = request.headers['x-amz-meta-md5chksum']
                         ? new Buffer(request.headers['x-amz-meta-md5chksum'],
                         'base64').toString('hex') : null;
+                    const ingestVersion = request.headers['x-amz-version-id'];
```

Comment:
It looks like the same header was just put into a variable here?

Reply:
Yeah, thank you. It must have got lost in my commits for some reason. I just force pushed the correct one.